byob/modules/ransom.py

```diff
@@ -15,6 +16,7 @@
 # packages
 import Crypto.PublicKey.RSA
```

Comment:
I just noticed, `Crypto.PublicKey.RSA` needs to updated to `Cryptodome.PublicKey.RSA` - if you could integrate this into your next commit I would appreciate it.